jupyterlab_gitsync/jupyterlab_gitsync/tests/nbinit_helper_test.py

```diff
@@ -0,0 +1,90 @@
+import unittest
+import subprocess
+from jupyterlab_gitsync.nb_handlers import NotebookInitHandler
+
+def make_setup():
+  subprocess.call(['mkdir', 'test_files'], cwd='.')
+  subprocess.call(['mkdir', 'test_files/.sync_cache'], cwd='.')
+  file_path = 'test_files/init_cache_test.ipynb'
+
+  file_contents = '{                            \n\
+                     "cells": [                 \n\
+                      {                         \n\
+                       "cell_type": "code",     \n\
+                       "execution_count": null, \n\
+                       "metadata": {},          \n\
+                       "outputs": [],           \n\
+                       "source": []             \n\
+                      }                         \n\
+                     ],                         \n\
+                     "metadata": {              \n\
+                      "kernelspec": {           \n\
+                       "display_name": "",      \n\
+                       "name": ""               \n\
+                      },                        \n\
+                      "language_info": {        \n\
+                       "name": ""               \n\
+                      }                         \n\
+                     },                         \n\
+                     "nbformat": 4,             \n\
+                     "nbformat_minor": 4        \n\
+                    }'
+
+  with open(file_path, 'w+') as file:
+    file.write(file_contents)
+
+def remove_setup(): 
```

Comment:
Use the tearDown() method instead: https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDown